test/core_checkM_test.py

```diff
@@ -616,7 +856,7 @@ def test_local_method(self):
             }
         })
         out_contents = sorted(os.listdir(output_dir))
-        #self.assertEqual(out_contents, ['storage', 'lineage.ms', 'bins'])
+        # self.assertEqual(out_contents, ['storage', 'lineage.ms', 'bins'])
```

Comment:
The next set of files are all data files for new tests.

The last file is a new `run_test.sh` file with an extra command added to run the `checkM` installation tests to ensure correct checkM functioning.